kubernetes/keytransparency-deployment.yml

```diff
@@ -0,0 +1,98 @@
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: kt-server
+spec:
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        run: kt-server
+    spec:
+      containers:
+      - name: kt-server
+        image: us.gcr.io/transparent-keyserver/keytransparency-server
+        ports:
+          - containerPort: 8080
+            name: json-grpc
+        env:
+            - name: LOG_URL
```

Comment:
This seems to be the default (as in the [documentation](https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/)). But I've tried: 
` - name: LOG_URL value: trillian-log:8090` 
and get an error: `error: error converting YAML to JSON: yaml: line 24: mapping values are not allowed in this context
`